statsmodels/duration/tests/test_phreg.py

```diff
@@ -272,6 +272,20 @@ def test_summary(self):
         rslt = mod.fit()
         rslt.summary()
 
+        strata = np.kron(np.arange(50), np.ones(4))
+        mod = PHReg(time, exog, status, strata=strata)
+        rslt = mod.fit()
+        smry = rslt.summary()
+        msg = "3 strata dropped for having no events"
+        assert(msg in str(smry))
```

Comment:
numpy.testing `assert_`
plain assert are removed when byte compiling Python with -o or something like that